apis/metal3.io/v1alpha1/baremetalhost_validation.go

```diff
@@ -96,6 +103,10 @@ func validateBMCAccess(s BareMetalHostSpec, bmcAccess bmc.AccessDetails) []error
 		errs = append(errs, fmt.Errorf("BMC driver %s does not support secure boot", bmcAccess.Type()))
 	}
 
+	if s.Image != nil && s.Image.DiskFormat != nil && *s.Image.DiskFormat == "live-iso" && !bmcAccess.SupportsISOPreprovisioningImage() {
```

Comment:
just a nit, maybe for the consistency we could define a constant for "live-iso" in https://github.com/metal3-io/baremetal-operator/blob/6966480dece8aed3cbf1f14180406548f87ec796/apis/metal3.io/v1alpha1/baremetalhost_types.go#L99

Reply:
updated, thanks.